src/js/references/components/Official.tsx

```diff
@@ -20,6 +20,10 @@ const StyledReferenceOfficial = styled(Box)`
     }
 `;
 
+const StyledButton = styled(Button)`
+    min-width: 83px;
+`;
```

Comment:
Why not just include this in `StyledReferenceOfficial`? Also naming nitpick, should be something like `InstallButton`, generally we avoid `styled` unless it is the top level div for an otherwise usefully named component. Ex.`StyledReferenceOfficial` is a case where it is fine.